package.json

```diff
@@ -25,6 +25,8 @@
     "@wagmi/core": "^1.4.1",
     "axios": "^1.5.0",
     "chart.js": "^4.4.0",
+    "chartjs-adapter-dayjs-4": "^1.0.4",
+    "dayjs": "^1.11.10",
```

Comment:
would using date-fns be an option?

Reply:
yeah but I'm finding it harder to use
perhaps I just need time to learn the api